### DISCUSSION TITLE
过期：手册中的一点小问题

### DISCUSSION THREAD

**Discussion by Arxhexa** [hooke007/mpv_PlayKit#242] [回收]
有一些vo=gpu-next的参数写成了vo=gpu
比如 [这里](https://hooke007.github.io/official_man/mpv.html#:~:text=%E4%B8%8D%E5%9D%87%E5%8C%80%E6%80%A7%EF%BC%88-,%2D%2Dvo%3Dgpu%20%E7%8B%AC%E5%8D%A0,-%EF%BC%89) 后面的 [这里](https://hooke007.github.io/official_man/mpv.html#:~:text=%E9%99%8D%E4%BD%8E%E8%BE%93%E5%87%BA%E4%BA%AE%E5%BA%A6%EF%BC%88-,%2D%2Dvo%3Dgpu%20%E7%8B%AC%E5%8D%A0,-%EF%BC%89) [这里](https://hooke007.github.io/official_man/mpv.html#:~:text=%E9%BB%98%E8%AE%A4%E4%B8%BA%E5%BC%80%E3%80%82%EF%BC%88-,%2D%2Dvo%3Dgpu%20%E7%8B%AC%E5%8D%A0,-%EF%BC%8C%E6%B3%A8%E6%84%8F%20%2D%2Dvo%3Dgpu)
和原版手册有一定出入
不太确定其他地方是否还存在这样的情况，只看了下hdr映射曲线相关的部分

**Comment by hooke007** [comment]
感谢，中间有一次硬盘数据丢失，涉及到几个项目的commit未及时同步推送，后边也想不起来改了啥了

**Reply by hooke007** [reply]
已更新
